tests/certification/bindings/zeebe/command/config.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: zeebecommandbindingconfig
```

In looking at the other bindings, it looks like the metadata name value can include `-`. It might lend towards a nicer user experience seeing the separated words there. Not sure if there was a reason there were no `-`s present?
```suggestion
  name: zeebe-command-binding-config
```